pkg/promtail/scrape/scrape.go

```diff
@@ -15,15 +19,38 @@ type Config struct {
 	JobName                string                           `yaml:"job_name,omitempty"`
 	EntryParser            api.EntryParser                  `yaml:"entry_parser"`
 	PipelineStages         stages.PipelineStages            `yaml:"pipeline_stages,omitempty"`
+	JournalConfig          *JournalTargetConfig             `yaml:"journal,omitempty"`
 	RelabelConfigs         []*relabel.Config                `yaml:"relabel_configs,omitempty"`
 	ServiceDiscoveryConfig sd_config.ServiceDiscoveryConfig `yaml:",inline"`
 }
 
+// JournalTargetConfig describes systemd journal records to scrape.
+type JournalTargetConfig struct {
+	// Since holds the time after time.Now() to start retrieving records
```

Comment:
it not clear since Now()

Reply:
I'm trying to rephrase the comment but it seems that the Since field is pretty obtuse. Either it's zero and we read all records from the journal (or from the cursor if saved) or it's non-zero and its added to the current system time once the target is initialized and that determines the earliest record to accept. The cursor is still used in that case.

Should we just take this out? I'm not sure it's all that useful.

Reply:
I just took it out, I can't think of a use case for it. Saving the position in the positions file makes the field unnecessary.